syntax = "proto3";

package user;

service UserService {
    rpc GetUser (UserRequest) returns (UserResponse);
    rpc CreateUser (User) returns (UserResponse);
}

message UserRequest {
    int32 id = 1;
}

message User {
    int32 id = 1;
    string name = 2;
}

message UserResponse {
    int32 id = 1;
    string name = 2;
}


